scholia/app/templates/base.html

```diff
@@ -395,17 +395,23 @@
     UNION
     { wd:{{ q }} wdt:P644 [] . BIND("gene" AS ?aspect) }
     UNION
+    { wd:{{ q }} p:P31/ps:P31 wd:Q22325163 . BIND("complex" AS ?aspect) }
```

Comment:
Why not wdt:P31?

Reply:
Because when there is more than one statement with that property, then the RDF becomes less predictable.

Reply:
I do not understand. Would multiple statements not resolve in multiple wdt's and muliple (p/ps)'s?